conn.go

```diff
@@ -135,6 +136,15 @@ func newMultiFlusher(flushers ...flusher) flusher {
 	return &multiFlusher{flushers}
 }
 
+// Underlying connection state information.
+type ConnInfo struct {
```

Comment:
Should be named `ConnectionState`

Reply:
There is already ConnState so I decided to pick a different name to prevent possible confusion. Should I still rename it to ConnectionState?

Reply:
Gah, okay, makes sense